src/modules/menu/components/share/MenuShareEmbed.vue

```diff
@@ -1,15 +1,21 @@
 <template>
     <div class="menu-share-embed">
         <button
-            class="btn btn-light btn-sm embedded-button"
+            :class="{
+                'rounded-0': showEmbedSharing,
+                'rounded-top btn btn-light btn-sm embedded-button': true,
+            }"
             data-cy="menu-share-embed-button"
             @click="toggleEmbedSharing"
         >
             <FontAwesomeIcon :icon="`caret-${showEmbedSharing ? 'down' : 'right'}`" />
             <span class="ms-2">{{ $t('share_more') }}</span>
         </button>
         <CollapseTransition :duration="200">
-            <div v-show="showEmbedSharing" class="p-2 card border-light bg-light">
+            <div
+                v-show="showEmbedSharing"
+                class="p-2 rounded-0 rounded-bottom rounded-end card border-light bg-light"
+            >
```

Comment:
Those changes are obsolete with #614 they should be reverted in order to avoid merge conflict